src/Framework/OpenXmlBasePart.cs

```diff
@@ -81,6 +81,11 @@ internal void Load(OpenXmlPackage openXmlPackage, OpenXmlPart parent, Uri uriTar
             this._uri = uriTarget;
 
             // TODO: should we delay load?
+            if (!this.OpenXmlPackage.Package.PartExists(uriTarget))
+            {
+                throw new OpenXmlPackageException(string.Format("The part with URI target {0} does not exist in the package.", uriTarget));
+            }
```

Comment:
Ultimately, I'd like the SDK to have the same behavior as Word - if a file is missing a part, and it is possible to add the part with reasonable content (in some cases just an empty part), then we should do so.

Our priorities currently are:
- Get the build system / xunit tests squared away
- Office 2016 support
- Fix the validator
- following this, priority is TBD